### ISSUE TITLE
Setting unknown property: verbb\vizy\models\BlockType::layout

### ISSUE THREAD

**Issue by piotrpog** [verbb/vizy#13]
This error showed up when i tried to add new group to existing field.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.1](https://github.com/verbb/vizy/releases/tag/1.0.1)
